test/locators.spec.ts

```diff
@@ -24,9 +23,19 @@ describe('fill form', () => {
         'Wait Commands',
         'WebElement Commands',
       ],
+      file: 'resources/git.jpeg',
+    });
+  });
+  it('then should have a photo', async () => {
+    const files: number = await personalInformationPage.getFiles();
```

Comment:
let's use the hook functions (before, beforeEach, etc) to organize the actions executed by the test, and the `it`s only for the `expect`s